src/ZODB/scripts/analyze.py

```diff
@@ -60,8 +59,8 @@ def shorten(s, n):
 def report(rep):
     print("Processed %d records in %d transactions" % (rep.OIDS, rep.TIDS))
     print("Average record size is %7.2f bytes" % (rep.DBYTES * 1.0 / rep.OIDS))
-    print(("Average transaction size is %7.2f bytes" %
-           (rep.DBYTES * 1.0 / rep.TIDS)))
+    print("Average transaction size is %7.2f bytes" %
+          (rep.DBYTES * 1.0 / rep.TIDS))
```

Comment:
None of these conversions are done by hand, they are done by `pyupgrade`. It misses some of them. Since these missed expressions won't break anything I don't do a line by line scan to find missed ones, either.

Reply:
I updated this one + some of the surrounding ones to f-strings but it does not make them more readable.